frontend/src/views/AmendingLawArticleEditorMultiMod.vue

```diff
@@ -216,13 +216,20 @@ function handleUpdate(event: MouseEvent) {
 
     <form v-else class="grid grid-cols-1 gap-y-12">
       <div class="grid grid-cols-2 gap-x-40">
-        <RisDropdownInput
-          id="timeBoundaries"
-          v-model="timeBoundary"
-          label="Zeitgrenze"
-          :items="timeBoundaryItems"
-          @change="preview"
-        />
+        <div class="flex flex-col gap-6">
+          <!-- eslint-disable-next-line vuejs-accessibility/label-has-for -->
+          <label id="timeBoundariesLabel" class="ris-label2-regular"
```

Comment:
Todo: should not use a hardcoded ID, use `useElementId` instead. See [styleguide](https://digitalservicebund.atlassian.net/wiki/spaces/VER/pages/884310020/Frontend+conventions+and+patterns#Avoid-using-IDs.-Do-not-assume-them-to-be-human-readable-or-static.)